index.html

```diff
@@ -70,41 +78,49 @@ <h2>I’am a Software Developer</h2>
           <h1>My Recent Works</h1>
           <hr />
         </div>
-        <img src="SVG/recentworkimg.svg" alt="" class="recentworkimg" />
-        <div class="recentworkdescription">
-          <h1>Multi-Post Stories</h1>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard dummy text ever
-            since the 1500s, when an unknown printer took a standard dummy text.
-          </p>
-          <ul class="languages">
-            <li>css</li>
-            <li>html</li>
-            <li>bootstrap</li>
-            <li>Ruby</li>
-          </ul>
-          <button class="btn">See Project</button>
+        <div class="recentWork">
+          <img src="SVG/recentworkimg.svg" alt="" class="recentworkimg" />
+          <div class="recentworkdescription">
+            <h1>Multi-Post Stories</h1>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul class="languages">
+              <li>css</li>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button class="btn">See Project</button>
+          </div>
         </div>
         <div class="workcards">
           <div id="w1" class="cards">
-            <div class="cardDiscription">
-              <h2>Profesional Art Printing Data</h2>
-              <p>
-                A daily selection of privately personalized reads; no accounts
-                or sign-ups required. has been the industry's standard
-              </p>
-              <ul class="cardlist">
-                <li>html</li>
-                <li>bootstrap</li>
-                <li>Ruby</li>
-              </ul>
+            <div class="showbutton">
+              <div class="cardDiscription" id="w1cardDiscription">
+                <h2><span>Profesional Art Printing Data</span></h2>
+                <p>
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required. has been the industry's standard
+                </p>
+                <ul class="cardlist">
+                  <li>html</li>
+                  <li>bootstrap</li>
+                  <li>Ruby</li>
+                </ul>
+              </div>
```

Comment:
- For the project cards, their default look should be like this:

![image](https://user-images.githubusercontent.com/50721479/194116738-b52886cb-bdf2-41d3-a386-8351d81f6133.png)

- When the user hovers, it should become like this:

![image](https://user-images.githubusercontent.com/50721479/194116804-538d9bf8-332e-4493-bbad-3c98f78238de.png)

- [ ] Kindly make the necessary changes to make the cards as mentioned above 👍